community/examples/htcondor-pool.yaml

```diff
@@ -20,6 +20,8 @@ vars:
   deployment_name: htcondor-001
   region: us-central1
   zone: us-central1-c
+  network_name: htcondor-pool
```

Comment:
I believe that you're "promoting" (sub)network_name to deployment variable to enable being overridden at CLI, correct?

We have previously made choice to attach these variables at module level because they are not used by non-VPC modules except v4 Slurm and Packer (which I intend to address). i.e. if it's doesn't need to be a deployment variable, let's not make it one.

Reply:
Intended as a point of discussion - enabling CLI override may merit the change, but I'd like to understand the intent.

Reply:
You are correct on the motivation. There are three options I can think of:
1. Move to deployment vars so they can be overwritten, as I have done
2. Remove hard-coded naming of networks and just allow default to take place: named after deployment which in all cases is very close to actual name **(prefered now that I write these out)**. I would still leave them in the case of the packer example as there are already in deployment vars and needed there.
3. In cases where names are hardcoded as settings, use sed in the test to pre-modify the blueprint, [example](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/5bc086a1239519b1b27255c6915abc6d41a7ff11/tools/cloud-build/daily-tests/builds/batch-mpi.yaml#L59-L67)

Reply:
Discussed offline, went with case 2.